pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[project]
+name = "Rotary"
+version = "0.3.0"
+description = "Rotary is a snakemake-based scalable pipeline for hybrid assembly of microbial genomes using Nanopore and Illumina data."
```

Comment:
Change to the following?
"Rotary is a toolkit and workflow for managing genome assemblies with circular DNA elements. It contains modules for addressing common issues with circular assemblies as well as a scalable snakemake-based pipeline that integrates these modules with best practices for hybrid short/long read microbial genome assembly."

Reply:
@jmtsuji Addressed in https://github.com/rotary-genomics/rotary/pull/102